src/api/views/payment/checkout_webhook_view.py

```diff
@@ -0,0 +1,103 @@
+from django.utils import timezone
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.request import Request
+from rest_framework.parsers import JSONParser
+from rest_framework.permissions import AllowAny
+
+from src.api.models import LicencePlate
+from src.core.views import BackendResponse, _OriginAPIView
+from src.users.models import User
+from src.api.serializers import LoginSerializer, UsersSerializer, LicencePlateSerializer
+
+from src.core.views import _OriginAPIView
+
+import stripe
+
+# Set your secret key. Remember to switch to your live secret key in production.
+# See your keys here: https://dashboard.stripe.com/apikeys
+stripe.api_key = 'sk_test_51Lf1SsGRh96C3wQGfjc1BuPw2AhNPQpteJ0fz3JXRiD8QzpOb5nVKeNDSOKyLpfw6qcVUZ9936duVmrylnAqWf1t00kkRqidz1'
+endpoint_secret = 'whsec_cbc3c8904ec1b2bcd029776f6217f81b9d11da0c4c06b472b574b529c6cf220c'
+
+
+def complete_order(session: stripe.checkout.Session) -> BackendResponse:
+    metadata = session.metadata
+    if 'licence_plate' in metadata.keys() and 'user_id' in metadata.keys():
+        licence_plate = session.metadata['licence_plate']
+        user_id = session.metadata['user_id']
+    else:
+        return BackendResponse(['The required data to complete the order was not included in the session metadata.'],
+                               status=status.HTTP_400_BAD_REQUEST)
+
+    licence_plate: LicencePlate = LicencePlate.objects.get(user=User.objects.get(pk=user_id),
+                                                           licence_plate=licence_plate)
+
+    licence_plate.updated_at = timezone.now()
+
+    try:
+        licence_plate.save()
+        return BackendResponse(['Completed order'], status=status.HTTP_200_OK)
+    except:
+        return BackendResponse(['Failed to save new updated_at time.'], status=status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
Personnaly I think this is a little bit overkill :sweat_smile: for using a `try...catch...`-block for this, because normally this operation will always succeed, but indeed, better safe than sorry!